src/instructlab/defaults.py

```diff
@@ -11,12 +11,10 @@
 CONFIG_VERSION = "1.0.0"
 
 # Model families understood by ilab
-MODEL_FAMILIES = {"merlinite", "mixtral"}
+MODEL_FAMILIES = {"granite", "mixtral"}
```

Comment:
I think the family should be `mistral` instead of `mixtral`, since that is the name of the company that makes the mixtral model and the mistral model

thoughts? @nathan-weinberg

Reply:
I'm amenable to that for sure - @cdoern wdyt? since this was your original suggestion

Reply:
mixtral is right, its what is used in SDG as the "family"

we map `mistral: mixtral` so that both types of models use the same templates.

Reply:
Just because we use it elsewhere doesn't mean it's "right" though

Reply:
sure, but I believe changing this would breakages in other places given that they depend on the "mixtral" model family

Reply:
I would prefer if we could discuss/do this in a separate PR - I am going to leave it as is to reduce the risk of causing a regression - happy to file a followup issue